test4044.py

```
@@ -0,0 +1,102 @@
+# Disable SSL warnings
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+warnings.filterwarnings("ignore", category=UserWarning, module='bs4')
+
+def util_validate(match):
+    return "\n" in match or ' ' in match
+
+def find_urls_and_endpoints(domain):
+    global_matches = set()
+
+    try:
+        response = requests.get(domain, verify=False, timeout=5)
```

Ruff (0.14.10)

Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)